### PR TITLE
Tuned: init at 2.16.0 

### PR DESCRIPTION
**Pull request by pasqui23**
###### Motivation for this change
Tuned is a performance tuning daemon developed by red hat.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: nixos/modules/services/hardware/tuned.nix [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+let
+  cfg = config.services.tuned;
+  p = pkgs.tuned;
+in
+{
+  options.services.tuned = {
+    enable = lib.mkEnableOption "tuned, a performance tuning daemon";
+  };
+
+  config = lib.mkif cfg.enable {
```
Comment (aanderse) [inline nixos/modules/services/hardware/tuned.nix:11]: `systemd.tmpfiles.packages`?

Comment (pasqui23) [reply nixos/modules/services/hardware/tuned.nix]: ah didn't know of that

**Comment by ilya-fedin** [comment]
Any progress? :thinking:

**Comment by pasqui23** [comment]
Unfortunately no. Tuned's model is inherently imperative, its packaging is  highly rpm specific and I simply cannot run it.
None of this is unsormountable but combined with lack of time and the fact that I solved my battery woes without it means I won't work on this anymore.

**Pull request closed without merging**